src/altscore/borrower_central/model/verifications.py

```
@@ -0,0 +1,226 @@
+                f"/v1/verifications/{verification_id}",
+                headers=self.build_headers(),
+                timeout=120,
+            )
+            raise_for_status_improved(request)
+            return request.json()
+
+    @retry_on_401
+    async def retrieve_all(self):
+        with httpx.Client(base_url=self.altscore_client._borrower_central_base_url) as client:
+            request = client.get(
+                f"/v1/verifications",
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)